backend/src/exercise/patient-ticking.ts

```diff
@@ -86,43 +67,30 @@ function getDedicatedResources(
     state: ExerciseState,
     patient: Patient
 ): Catering {
+    const cateringTypes: Catering = {
+        notarzt: 0,
+        notSan: 0,
+        rettSan: 0,
+        san: 0,
+        gf: 0,
+        material: 0,
+    };
+
     if (!patient.isBeingTreated) {
```

Comment:
Maybe putting a comment here or changing it.
Right now there is a dependency as a patient can't be treated by material alone, but this choice shouldn't be done here probably (or at least be mentioned via comments). Right now this returns zero material, even if a patient has material.